src/Basic/NaturalTransformation.lidr

```diff
@@ -186,30 +187,38 @@ The code above is everything we need to define what a natural transformation is.
 >                      (commutativity natTrans1 a b f))
 >         (sym $ associativity cat2 _ _ _ _ (mapMor fun1 a b f) (component natTrans1 b) (component natTrans2 b))))))
 >
-> composeFunctorNatTrans :
+> naturalTransformationHorizontalComposition :
 >      (cat1, cat2, cat3 : Category)
 >   -> (fun1, fun2 : CFunctor cat1 cat2)
+>   -> (fun3, fun4 : CFunctor cat2 cat3)
```

Comment:
maybe this could be a good time to start introducing some implicits, what do you think?

Reply:
I don't have a good feeling for this yet, but it seems to be pretty safe to make objects implicit if they are the source or target of a morphism that is passed along too, since most code seems to just use `_` for those arguments anyway.